app/src/main/java/org/ole/planet/myplanet/ui/team/MyTeamsDetailFragment.kt

```diff
@@ -233,8 +238,9 @@ class MyTeamsDetailFragment : BaseNewsFragment() {
         llRv.visibility = View.GONE
         tab.setText(s)
         listContent.adapter = object : ArrayAdapter<RealmUserModel?>(requireActivity(), android.R.layout.simple_list_item_1, data) {
-            override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
-                var convertView = convertView
+            @SuppressLint("SetTextI18n")
```

Comment:
thought the fix was only for the `notifyDataSetChanged()` but we have  other fixes that can be done in differen pull requests